postprocessing/visualization/tools/plot_EnergyOutput.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Oct  4 21:18:55 2023
+
+@author: fkutschera
+"""
+import matplotlib
+matplotlib.use('Agg')
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import argparse
+import sys
+import os
+
+description = '''Plot SeisSol energy output(s). 
+You can run the script with the default (Moment rate [MR]) or select multiple energy outputs available from a SeisSol simulation.
+You can also compare the energy outputs of different simulations.)'''
+parser = argparse.ArgumentParser(description=description)
+parser.add_argument("Input", nargs="+", metavar=("file"),
+                    help="energy.csv file(s) from your SeisSol simulation. Multiple input files possible.")
+parser.add_argument("-d", "--Data", type=str, required=False, nargs="+", metavar=("variable"), default=["MR"], 
+                    help="Data to plot. Default: Moment rate (MR). Other available options are Gravitational energy (GE), Acoustic energy (AE), Acoustic kinetic energy (AKE), Elastic kinetic energy (EKE), Elastic energy (EE), Total frictional work (TFW), Static frictional work (SFW), Seismic moment (SM), Plastic moment (PM)")
+parser.add_argument("--xlim", nargs="+", type=float, help="xmin xmax")
+parser.add_argument("--ylim", nargs="+", type=float, help="ymin ymax")
+parser.add_argument("--xlabel", default="Time [s]", type=str, help="Choose xlabel")
+parser.add_argument("--ylabel", default="", type=str, help="Choose ylabel")
+parser.add_argument("--dpi", default=300, type=int, help="Dots per Inch")
+parser.add_argument("-o", "--Output", help='Name of the output file', default='energy_output.png')
+args = parser.parse_args()
+
```

Comment:
Allow wildcards file path
```suggestion

files = [] 
for _f in args.Input:
    files.extend(glob.glob(_f))
```